app/src/main/java/com/portfolio/tasky/entry/fragments/LoginFragment.kt

```diff
@@ -51,21 +53,18 @@ class LoginFragment : Fragment(), FragmentInflater by FragmentInflaterImpl(), Te
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        setToolbar()
         setObservers()
-        viewBinding.btnLogin.setOnClickListener {
-            val email = viewBinding.etEmail.subLayout.etInput.text.toString()
-            val password = viewBinding.etPassword.subLayout.etInput.text.toString()
 
-            lifecycleScope.launch(Dispatchers.IO + coroutineExceptionHandler){
-                viewModel.login(AuthenticationRequest(email, password))
-            }
+        (activity as EntryActivity).hideFAB()
 
-            viewModel.loginObserver.observe(viewLifecycleOwner){
-                if(it != null){
-                    Toast.makeText(activity, "Token ${it.token}", Toast.LENGTH_SHORT).show()
-                }
-            }
-        }
+        viewBinding.btnLogin.setOnClickListener(this)
+        viewBinding.tvRegister.setOnClickListener(this)
+    }
+
+    private fun setToolbar() {
+        setFragmentManager(activity?.supportFragmentManager as FragmentManager)
+        (activity as EntryActivity).setTitle((activity as EntryActivity).getString(R.string.welcome_back))
```

Comment:
I find this line hard to read due to all the conversions. You could make it a bit more readable like this:
```
val parentActivity = requireActivity() as EntryActivity
parentActivity.setTitle(requireContext().getString(R.string.welcome_back))
```

Reply:
Done